packages/api/src/controllers/cropVarietyController.js

```diff
@@ -68,10 +68,23 @@ const cropVarietyController = {
   createCropVariety() {
     return async (req, res, next) => {
       try {
-        const { crop_id } = req.body;
+        const { crop_id, crop_variety_name, farm_id } = req.body;
         const [relatedCrop] = await CropModel.query()
           .context({ showHidden: true })
           .where({ crop_id });
+        if ((farm_id, crop_id, crop_variety_name)) {
```

Comment:
@Duncan-Brain `||` ? `&&`?

Reply:
@SayakaOno @kathyavini 

yah I messed up my eyes arent working and im rushing

Reply:
Obviously mine too because I didn't see that when I reviewed for merge!

Thank you @SayakaOno!!

Reply:
lol that would have broke my half of it -- @SayakaOno @kathyavini please check again

Reply:
LOL!!! ❤️